mirdata/utils.py

```diff
@@ -71,6 +74,12 @@ def validator(dataset_index, data_home, silence=False):
                         invalid_checksums[track_id] = []
                     invalid_checksums[track_id].append(local_path)
 
+    return missing_files, invalid_checksums
+
+
+def validator(dataset_index, data_home, dataset_path, silence=False):
+    missing_files, invalid_checksums = check_index(dataset_index, data_home, silence)
```

Comment:
before running `check_index`, first call `check_validated`, and skip running if it's valid.

Reply:
should be changed now.